src/Test/MakerTestRunner.php

```diff
@@ -0,0 +1,209 @@
+
+        if (!file_exists($path)) {
+            throw new \Exception(sprintf('Cannot find file "%s"', $path));
+        }
+
+        if (is_file($path)) {
+            $this->filesystem->copy($path, $this->getPath($destination), true);
+
+            return;
+        }
+
+        // handle a directory copy
+        $finder = new Finder();
+        $finder->in($path)->files();
+
+        foreach ($finder as $file) {
+            $this->filesystem->copy($file->getPathname(), $this->getPath($file->getRelativePathname()), true);
+        }
+    }
+
+    public function renderTemplateFile(string $source, string $destination, array $variables): void
+    {
+        $twig = new Environment(
+            new FilesystemLoader(__DIR__.'/../../tests/fixtures')
+        );
+
+        $rendered = $twig->render($source, $variables);
+
+        $this->filesystem->mkdir(\dirname($this->getPath($destination)));
+        file_put_contents($this->getPath($destination), $rendered);
+    }
+
+    public function getPath(string $filename): string
+    {
+        return $this->environment->getPath().'/'.$filename;
+    }
+
+    public function readYaml(string $filename): array
+    {
+        return Yaml::parse(file_get_contents($this->getPath($filename)));
+    }
+
+    public function getExecutedMakerProcess(): MakerTestProcess
+    {
+        if (!$this->executedMakerProcess) {
+            throw new \Exception('Maker process has not been executed yet.');
+        }
+
+        return $this->executedMakerProcess;
+    }
+
+    public function modifyYamlFile(string $filename, \Closure $callback)
+    {
+        $path = $this->getPath($filename);
+        $manipulator = new YamlSourceManipulator(file_get_contents($path));
+
+        $newData = $callback($manipulator->getData());
+        $manipulator->setData($newData);
+
+        file_put_contents($path, $manipulator->getContents());
+    }
+
+    public function runConsole(string $command, array $inputs, string $arguments = '')
+    {
+        $process = $this->environment->createInteractiveCommandProcess(
+            $command,
+            $inputs,
+            $arguments
+        );
+
+        $process->run();
+    }
+
+    public function replaceInFile(string $filename, string $find, string $replace, bool $allowNotFound = false): void
+    {
+        $this->environment->processReplacement(
+            $this->environment->getPath(),
+            $filename,
+            $find,
+            $replace,
+            $allowNotFound
+        );
+    }
+
+    public function configureDatabase(bool $createSchema = true): void
+    {
+        $this->replaceInFile(
+            '.env',
+            'postgresql://symfony:ChangeMe@127.0.0.1:5432/app?serverVersion=13&charset=utf8',
+            getenv('TEST_DATABASE_DSN')
+        );
+
+        // Flex includes a recipe to suffix the dbname w/ "_test" - lets keep
+        // things simple for these tests and not do that.
+        $this->replaceInFile(
```

Comment:
Would be nice to have removeFromFile() or removeInFile() method which does the replacement internally.

It's easier to read